debugger.jpda/src/org/netbeans/modules/debugger/jpda/expr/EvaluatorVisitor.java

```diff
@@ -1992,7 +1992,7 @@ public Mirror visitIdentifier(IdentifierTree arg0, EvaluationContext evaluationC
                 identifierPath = getCurrentPath();
             }
             elm = evaluationContext.getTrees().getElement(identifierPath);
-            if (elm instanceof TypeElement && ((TypeElement) elm).asType() instanceof ErrorType) {
+            if (elm instanceof TypeElement && ((TypeElement) elm).asType().getKind() == TypeKind.ERROR) {
```

Comment:
ClassTypes are both DeclaredType and ErrorType in current JDK 11. This presumably would be merged into nb-javac at some point.